test/fixtures/responses/index.js

```diff
@@ -31,26 +34,37 @@ module.exports = {
     notValidated: require('./get-transaction-not-validated.json'),
     suspendedPaymentCreation:
       require('./get-transaction-suspended-payment-create.json'),
+    SuspendedPaymentCreationIOU:
+      require('./get-transaction-suspended-payment-create-iou.json'),
     suspendedPaymentCancellation:
       require('./get-transaction-suspended-payment-cancellation.json'),
     suspendedPaymentExecution:
-      require('./get-transaction-suspended-payment-execution.json')
+      require('./get-transaction-suspended-payment-execution.json'),
+    suspendedPaymentExecutionSimple:
+      require('./get-transaction-suspended-payment-execution-simple.json')
   },
   getTransactions: require('./get-transactions.json'),
+  getTransactionsOne: require('./get-transactions-one.json'),
   getTrustlines: require('./get-trustlines.json'),
+  getTrustlinesAll: require('./get-trustlines-all.json'),
   getLedger: {
     header: require('./get-ledger'),
     full: require('./get-ledger-full'),
-    withSettingsTx: require('./get-ledger-with-settings-tx')
+    withSettingsTx: require('./get-ledger-with-settings-tx'),
+    withStateAsHashes: require('./get-ledger-with-state-as-hashes')
   },
-  prepareOrderCancellation: require('./prepare-order-cancellation.json'),
   prepareOrder: {
     buy: require('./prepare-order.json'),
     sell: require('./prepare-order-sell.json'),
-    expiration: require('./prepare-order-expiration')
+    expiration: require('./prepare-order-expiration'),
+    cancellation: require('./prepare-order-cancellation.json'),
```

Comment:
`prepareOrder` and `prepareOrderCancellation` should be separate namespaces because they are separate methods